controller/planner_backlog_blackbox_test.go

```diff
@@ -97,7 +96,7 @@ func (rest *TestPlannerBacklogBlackboxREST) setupPlannerBacklogWorkItems() (test
 			SpaceID:    testSpace.ID,
 			Path:       append(parentIteration.Path, parentIteration.ID),
 			State:      iteration.IterationStateStart,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c